service/src/test/org/apache/hive/service/server/TestHS2HttpServerPamConfiguration.java

```diff
@@ -48,6 +48,7 @@
   private static HiveConf hiveConf = null;
   private static String keyStorePassword = "123456";
   private static String keyFileName = "myKeyStore";
+  private static String keyStoreType = "jks";
```

Comment:
You might want this to be `KeyStore. getDefaultType()` depending on the JDK being used? I think in most cases this should be ok though.

Reply:
Will fix